src/content/docs/style-guide/formatting/code-examples.mdx

```diff
@@ -183,20 +208,11 @@ These sections break down when to use specific types of code formatting.
     </dd>
   </Collapser>
 
-  <Collapser
-    id="add-syntax-higlighting"
-    title="Add syntax highlighting"
-  >
-    To add syntax highlighting to a code block, add `class="highlight"` to the `<pre>` tag. The syntax highlighter tries to auto-detect the language. If it's not working as expected, ensure the language is supported and specify the language by adding the a `lang-LANG-NAME` class (for example: `lang-html)`.
-
-    View a [full list of supported languages for syntax highlighting](/docs/new-relic-only/advanced-style-guide/design-specifications/supported-languages-syntax-highlighting). Element colors and themes are fully customizable.
-  </Collapser>
-
   <Collapser
     id="link-in-code-blocks"
     title="Link to other docs"
   >
-    You can add `<a href>` tags inside a `<pre>` blocks, just as you would any other content. However, make sure to use them judiciously since they can be easy to miss. A good example is the [Infrastructure config file template](/docs/infrastructure/new-relic-infrastructure/configuration/infrastructure-config-file-template-newrelic-infrayml), where each config element links to the relevant section of the main config doc.
+    You can add `<a href>` tags inside a of your code blocks, just as you would any other content. However, make sure to use them judiciously since they can be easy to miss. A good example is the [Infrastructure config file template](/docs/infrastructure/new-relic-infrastructure/configuration/infrastructure-config-file-template-newrelic-infrayml), where each config element links to the relevant section of the main config doc.
```

Comment:
Remove the "a" after inside, or reword the sentence.

Reply:
@urbiz-nr, good catch! I'll fix this!